src/opencl-crypt/aes.h

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2002-2016 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
```

Comment:
Can we remove everything but the verify for now?

Reply:
So have only the files needed to generate and run cl_ed25519_verify (OpenCL variant of ed25519_verify) ?